packages/components/src/components/post-back-to-top/post-back-to-top.tsx

```diff
@@ -60,17 +60,43 @@ export class PostBackToTop {
   componentDidLoad() {
     window.addEventListener('scroll', this.handleScroll, false);
 
-    this.translateY = window
-      .getComputedStyle(this.host)
-      .getPropertyValue('--post-floating-button-translate-y');
+    // Get the back-to-top button top postiion
```

Comment:
Please move the following calculations and sub functions in a dedicated class member (private function).
Also, are we 100% sure, we need that much JS to solve the issue. I would prefer to use as much CSS as possible to do the same thing and only use JS, if we have no other possibility.